aiomisc/backoff.py

```diff
@@ -96,6 +115,9 @@ async def run() -> Any:
                     except Exception as e:
                         last_exc = e
                         raise
+                    finally:
+                        delta += monotonic()
+                        statistic.sum_time += delta
```

Comment:
Why there is no `done` key like in the other statistics?
It will allow to approximate current "in-fly" processes.